Sources/Target/Linux/Process.cpp

```diff
@@ -125,7 +125,7 @@ ErrorCode Process::wait() {
 
   while (!_threads.empty()) {
     tid = blocking_waitpid(-1, &status, __WALL);
-    DS2LOG(Debug, "wait tid=%d status=%#x", tid, status);
+    DS2LOG(Debug, "wait: tid %d %s", tid, Stringify::WaitStatus(status));
```

Comment:
Couple of things here:
* you should put this log after the check at line 130 (`if (tid <= 0)`);
* use `PRI_PID` for the formatter of `tid`;
* you can remove the `wait: ` prefix.